gitlabform/processors/project/tags_processor.py

```diff
@@ -33,13 +36,20 @@ def _process_configuration(self, project_and_group: str, configuration: dict):
                             elif "user_id" in config:
                                 user_ids.add(config["user_id"])
                             elif "user" in config:
-                                user_ids.add(self.gitlab._get_user_id(config["user"]))
+                                users_by_username = self.gl.users.list(
```

Comment:
I believe we're looking for a single user so that wr can find their ID. Lets change the variable name to be more appropriate. Something like `user` or `gitlab_user`.

Reply:
Changed to `gitlab_user`